.github/workflows/golangci-lint.yml

```diff
@@ -13,13 +13,13 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-go@v4
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
         with:
-          go-version: '1.19'
+          go-version: '1.20'
```

Comment:
https://github.com/nspcc-dev/neofs-node/commit/4de9938c0375a04c8577b2168f32d8f9c764f25f ?

Reply:
Good one, added that.